<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>971</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_1" stretch="9,0,0,0">
     <item>
      <widget class="ModelViewer" name="glMain"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_1">
       <item>
        <widget class="QPushButton" name="btnTest">
         <property name="text">
          <string>Test</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoad">
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnNormalize">
         <property name="text">
          <string>Normalize</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCurv">
         <property name="text">
          <string>Curvature</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDetour">
         <property name="text">
          <string>Detour</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCut">
         <property name="text">
          <string>Cut</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDrawCir">
         <property name="text">
          <string>Fill</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="edition">
         <property name="text">
          <string>Edition</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="selection">
         <property name="text">
          <string>Selection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showPlate">
         <property name="text">
          <string>ShowPlate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showSkel">
         <property name="text">
          <string>Show Skel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbbSelection"/>
       </item>
       <item>
        <widget class="QComboBox" name="cmbbObject"/>
       </item>
       <item>
        <widget class="QSpinBox" name="edgeNum">
         <property name="minimum">
          <number>3</number>
         </property>
         <property name="maximum">
          <number>16</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="btnOutput">
         <property name="text">
          <string>Output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoadSkel">
         <property name="text">
          <string>Load Skel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCutSkel">
         <property name="text">
          <string>Cut Skel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPushConnector">
         <property name="text">
          <string>Push Conn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPullConnector">
         <property name="text">
          <string>Pull Conn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPushFace">
         <property name="text">
          <string>Push Face</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPullFace">
         <property name="text">
          <string>Pull Face</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnPutConnector">
         <property name="text">
          <string>Put Conn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDig">
         <property name="text">
          <string>Dig</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSwell">
         <property name="text">
          <string>Swell</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cut_s2">
         <property name="text">
          <string>Special Cut 2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_cut_s3">
         <property name="text">
          <string>Special Cut 3</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QPushButton" name="btnTest2">
         <property name="text">
          <string>Test2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnIso">
         <property name="text">
          <string>Isolate Body</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnIso_2">
         <property name="text">
          <string>Isolate Body2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnShell">
         <property name="text">
          <string>Make Shell</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCut_s">
         <property name="text">
          <string>Special Cut</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDecompose">
         <property name="text">
          <string>Decompose</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnDecompose_2">
         <property name="text">
          <string>Decompose2</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="sig1">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ModelViewer</class>
   <extends>QOpenGLWidget</extends>
   <header>modelviewer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
